network-api/networkapi/buyersguide/templates/about/methodology.html

```diff
@@ -45,19 +47,65 @@ <h3 class="p-heading mt-3 mb-1" id="product-selection">
 
       <p class="body">
         {% blocktrans trimmed %}
-        There are 76 products in the 2019 version of our <em>*Privacy Not Included</em> buyer’s guide. These products
-        fit into six categories — Toys & Games, Smart Home, Entertainment, Wearables, Health & Exercise, and Pets.
+        There are 136 products in the 2020 version of our <em>*Privacy Not Included</em>
+        buyer’s guide. These products fit into seven categories — Toys & Games, Smart Home,
+        Home Office (*new), Entertainment, Wearables, Health & Exercise, and Pets.
```

Comment:
is `*new` supposed to be a footnote, or typeset as superscript/subscript? It seems a bit odd to have that be regular text @caltrider